eng/Version.Details.xml

```diff
@@ -75,9 +75,9 @@
       <SourceBuild RepoName="emsdk" ManagedOnly="true" />
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="9.0.0-alpha.1.24466.2">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="9.0.0-alpha.1.24467.3">
       <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
-      <Sha>0cbbe19fd2e94912ae9ab5dd15431ee5193d398c</Sha>
+      <Sha>3e0ff2ba7689642831daf50abf4e85944d58d393</Sha>
```

Comment:
This commit includes the latest intellisense source build addition: https://github.com/dotnet/source-build-reference-packages/commits/release/9.0/